### ISSUE TITLE
Enable radio buttons to select which content can be seen on library.kiwix.org

### ISSUE THREAD

**Issue by Popolechien** [kiwix/libkiwix#1301]
The current selection for categories is that we chose to display only one category:

<img width="265" height="421" alt="Image" src="https://github.com/user-attachments/assets/a9df301a-81f2-47c9-960f-256cc46db4ca" />

But with Maps and Youtube, we now are entering a period where there's a lot of repetitive clutter that can not be easily parsed. It would be better to be able which categories should be displayed, via radio buttons.

<img width="154" height="260" alt="Image" src="https://github.com/user-attachments/assets/8dfaa3fd-94cf-44b5-9af2-f8f3a82e01b5" />

**Comment by rgaudin** [comment]
@Popolechien, as for @kelson42, please refrain from adding `good first issue` label.

A good first issue is a ticket that is (surprise) as the name suggest, a good first-contribution to this repo: the ticket is clear, has no pending question, implementation is either already discussed or trivial.

In general, only maintainers should set it as subtle details can change the nature of ticket drastically.
